src/test/java/Homework16.java

```diff
@@ -14,10 +14,13 @@ public void Navigation(){
 
         WebDriver driver = new ChromeDriver();
         driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
-        String url = "https//:bbb.testpro.io";
+
+        String url = "https://bbb.testpro.io";
         driver.get(url);
+
         WebElement registrationLink = driver.findElement(By.cssSelector("#hel"));
         registrationLink.click();
+
         String registrationUrl = "https://bbb.testpro.io/registration.php";
         Assert.assertEquals(driver.getCurrentUrl(), registrationUrl);
```

Comment:
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

For example  `public void navigation()`